Tiltfile

```diff
@@ -39,12 +43,20 @@ VALID_MODES = ["path_only", "path_with_auth"]
 # Check if the MODE is valid
 if MODE not in VALID_MODES:
     fail("Invalid MODE: '{}'. Allowed values are {}. Please set a valid MODE.".format(MODE, VALID_MODES))
+
 # --------------------------------------------------------------------------- #
 #                                PATH Service                                 #
 # --------------------------------------------------------------------------- #
 # 1. PATH Service                                                             #
 # --------------------------------------------------------------------------- #
 
+# The folder containing PATH's local configuration files.
+PATH_LOCAL_DIR = LOCAL_DIR + "/path"
+# The configuration file for PATH.
+PATH_LOCAL_CONFIG_FILE = PATH_LOCAL_DIR + "/.config.yaml"
```

Comment:
Ditto re changing the location of config file for PATH.

Reply:
See above comment. IMO it is an improvement.